TrafficSimulator/Assets/RoadGenerator/Core/Editor/RoadSystemEditor.cs

```diff
@@ -11,12 +11,16 @@ public class RoadSystemEditor : Editor
             private SerializedProperty _drivingSide;
             private SerializedProperty _spawnRoadsAtOrigin;
             private SerializedProperty _showGraph;
+            private SerializedProperty _defaultTrafficLight;
+            private SerializedProperty _defaultTrafficLightController;
         #endregion
         public void OnEnable()
```

Comment:
The roadsystem prefab doesn't have Default traffic light prefab and controller prefab assigned. It should be assigned.